movies/movies/WebViewController.swift

```diff
@@ -0,0 +1,88 @@
+// WebViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+//
+//  WebViewController.swift
+//  movies
+//
+//  Created by angelina on 27.10.2022.
+//
+import UIKit
+import WebKit
+
+/// Экран вебвью
+final class WebViewController: UIViewController {
+    // MARK: - Private Enum
+
+    private enum Constants {
+        static let urlShemeHostPath = "https://www.themoviedb.org/movie/"
+        static let urlFragment = "#play="
+    }
+
+    // MARK: - Private Visual Components
+
+    private lazy var webView: WKWebView = {
+        let webView = WKWebView()
+        webView.uiDelegate = self
+        webView.navigationDelegate = self
+        webView.translatesAutoresizingMaskIntoConstraints = false
+        return webView
+    }()
+
+    // MARK: - Private property
+
+    private let service = Service()
```

Comment:
```suggestion
    private let service = Service.shared
```
 по 10 раз создаешь сервис, создай его 1 раз и прокидывай везде либо как синглтон сделай